source/kraken/metrics.cpp

```diff
@@ -151,6 +151,48 @@ Metrics::Metrics(const boost::optional<std::string>& endpoint, const std::string
                                              .Labels({{"coverage", coverage}})
                                              .Register(*registry)
                                              .Add({}, create_exponential_buckets(0.5, 2, 10));
+
+    this->retrieve_rt_message_duration_histogram = &prometheus::BuildHistogram()
+                                                        .Name("kraken_retrieve_rt_message_duration_seconds")
+                                                        .Help("duration of RT messages retrieval from RabbitMQ")
+                                                        .Labels({{"coverage", coverage}})
+                                                        .Register(*registry)
+                                                        .Add({}, create_exponential_buckets(0.5, 2, 10));
+
+    this->retrieved_rt_message_number_histogram = &prometheus::BuildHistogram()
+                                                       .Name("kraken_retrieve_rt_message_number")
+                                                       .Help("number of RT messages retrieved from RabbitMQ")
+                                                       .Labels({{"coverage", coverage}})
+                                                       .Register(*registry)
+                                                       .Add({}, create_exponential_buckets(0.5, 2, 10));
+
+    this->applied_rt_entity_number_histogram = &prometheus::BuildHistogram()
```

Comment:
I think a [Counter](https://prometheus.io/docs/concepts/metric_types/#counter) is more appropriate than an histogram here

Reply:
Let's discuss that, but I'm not sure that it's appropriate here: we want to track the number of applied entities in a batch, so it's not monotonic IMO.

Reply:
:+1: